src/helpers/MarkdownHelpers.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Groups an array of changes by their 'type' property
+ * @param object The input object
+ * @returns The grouped object
+ */
```

Comment:
I think you meant to put this comment just above the `groupChangesByType` function